sdcm/nemesis.py

```diff
@@ -1710,6 +1710,93 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def streaming_err_and_rebuild(self, task='decommission'):
```

Comment:
change the name as it's not clear

Reply:
Renamed it to `break_streaming_task_and_rebuild`